maquetado/setup/scss/navbar.scss

```diff
@@ -21,3 +21,7 @@
   padding: 20px;
   font-size: $text-big;
 }
+
+@media only screen and (max-width: 1024px) {
+  .navbar-title{display: none;}
```

Comment:
Even if it is one single style line, make it in multiple lines and add spaces

  .navbar-title {
    display: none;
  }